jsonrpc/src/methods/history.json

```diff
@@ -115,6 +115,21 @@
       "$ref": "#/components/contentDescriptors/OfferResult"
     }
   },
+  {
+    "name": "portal_historyWireOffer",
+    "summary": "Send an OFFER request with given ContentKeys, to the designated peer and wait for a response. Requires the content keys to be stored locally. Returns the content keys bitlist upon successful content transmission or empty bitlist receive.",
```

Comment:
It is not fully clear to me from the explanation whether the call should also do the uTP transfers or not (and wait for their result or not).

The result type is just `OfferResult`, which is practically the bitlist, so the result of the offer (the returned accept).

But it does say `Requires the content keys to be stored locally`. I assume this should be: `Requires the content to be stored locally`.  And that would mean that it needs to content because it also needs to send it.

So please clarify what this call is supposed to do.

If the call is only supposed to do the offer/accept (without utp part) then I think this should get a `debug_portal` prefix (we probably should revise if more should then go in the `debug` prefix, but that's another issue).

If the call is supposed to do offer/accept/utp, then I'm not 100% sure, but it still feels like a `debug` call because of the fact that you already need to have stored the content locally.